tests/unit/test_config_node.py

```diff
@@ -153,7 +153,7 @@ def test_add_cylc_install_to_rose_conf_node_opts(rose_conf, cli_conf, expect):
     expect_opt += ' (cylc-install)'
 
     assert result.comments == [(
-        f' Config Options \'{expect_opt}\' from CLI '
+        f' Config Options {expect_opt!r} from CLI '
```

Comment:
In `tox.ini` you said this doesn't work at Python 3.7, but the tests are passing?

Reply:
Comment misleading - I've reworded it:
Flake 8 bugbear only checks for this issue at Python > 3.7

Reply:
Happy to disable it if we can't be bothered with it but the fact it isn't checked at Python 3.7 doesn't seem like a good reason